vllm/v1/core/block_pool.py

```diff
@@ -166,6 +169,9 @@ def __init__(
         self.enable_kv_cache_events = enable_kv_cache_events
         self.kv_event_queue: list[KVCacheEvent] = []
 
+        # KV cache metrics collector
```

Comment:
Comment doesn't add anything - we're in the KV cache code, and the variable is "metrics collector"